qcdb/qcvars/psivardefs.py

```diff
@@ -104,6 +104,10 @@ def wfn_psivars():
         args=['CUSTOM SCS-MP2 TOTAL ENERGY', 'CUSTOM SCS-MP2 CORRELATION ENERGY', 'HF TOTAL ENERGY'],
         coeff=[-1, 1, 1]))
 
+    pv0.extend(_solve_in_turn(
+        args=['CUSTOM SCS-MP2(N) TOTAL ENERGY', 'CUSTOM SCS-MP2(N) CORRELATION ENERGY', 'HF TOTAL ENERGY'],
+        coeff=[-1, 1, 1]))
+
```

Comment:
this shouldn't be here. only SCS-MP2 gets a custom and that's to distinguish SCS-MP2 the method (with fixed coeff) from CUSTOM SCS-MP2 the user-def combo